project/AkkaDependency.scala

```diff
@@ -14,7 +14,7 @@ object AkkaDependency {
   // else if akka version is "default", then the hard coded default will be used (jenkins doesn't allow empty values for config axis)
   // else if akka.version is anything else, then the given version will be used
 
-  val defaultAkkaVersion = "2.5.23"
+  val defaultAkkaVersion = "2.5.25"
```

Comment:
We usually bump the Akka version only if strictly needed.

Reply:
```suggestion
  // Updated only when needed, https://github.com/akka/akka/issues/26985
  val defaultAkkaVersion = "2.5.25"
```